### DISCUSSION TITLE
Can we also remove Recorder as part of HASPone?

### DISCUSSION THREAD

**Discussion by IIAIronWolf** [HASwitchPlate/HASPone#19] [Q&A] [answered]
Prior to HASPone, the recorder component was required to be implemented in our configuration.yaml files. Unless configured properly (and the old documentation didn't say anything about how to configure it), recorder can cause a lot of issue with system performance and SD card degradation for those running Home Assistant on a Raspberry Pi.

My question: Is the recorder component no longer needed thanks to HASPone?

**Comment by aderusha** [comment, marked as answer]
I think this should be removable.  I just pulled it out of my test install and it seems like everything still works.  There was a heavy dependency on "helper objects" retaining state between reboots for HASP to work.  Blueprints don't support helper objects, so I couldn't rely on that functionality with the new release (now I (ab)use MQTT for that).

This hasn't been extensively tested but I think you should be clear to comment that out if you don't want the rest of the functionality it provides.
